config.json

```diff
@@ -3,11 +3,13 @@
   { "target": "filecoin-project/go-fil-commp-hashhash" },
   { "target": "filecoin-project/go-hamt-ipld" },
   { "target": "ipfs/go-block-format" },
+  { "target": "ipfs/go-cid" },
   { "target": "ipfs/go-detect-race" },
   { "target": "ipfs/go-ds-badger" },
   { "target": "ipfs/go-ds-bitcask" },
   { "target": "ipfs/go-ds-measure" },
   { "target": "ipfs/go-ds-pebble" },
+  { "target": "ipfs/go-ds-s3" },
```

Comment:
These tests need access to a fake S3 service (we use a docker container) and a go-ipfs binary.

See https://github.com/ipfs/go-ds-s3/blob/master/.circleci/config.yml

I'm not sure how best to fix this. Maybe we add something to this config so we can disable the tests action for some repos? That way we can implement custom tests when necessary?

Reply:
This brings us back to templating, right? cc @mvdan

Reply:
Note: I have a feeling we may need to switch to a yaml config to make that happen (at least without going insane). That is, I expect we'll want to be able to "patch-in" steps.

Reply:
Yes, but I guess there are a few options:

1. We could just have some flags. In this case: "needs-s3: true".
2. Arbitrary yaml (in which case I'd switch this to config.yml so we can more easily configure it).

Option 1 is likely simpler and a better option at the moment.

Also, thinking about this, we _don't_ need the go-ipfs binary here. We only needed that for the plugin test, which should honestly be a separate action.